JustSaying/AwsTools/ErrorQueue.cs

```diff
@@ -24,7 +24,7 @@ public ErrorQueue(RegionEndpoint region, string sourceQueueName, IAmazonSQS clie
         {
             return new Dictionary<string, string>
             {
-                { SQSConstants.ATTRIBUTE_MESSAGE_RETENTION_PERIOD , queueConfig.ErrorQueueRetentionPeriodSeconds.ToString(CultureInfo.InvariantCulture)},
+                { SQSConstants.ATTRIBUTE_MESSAGE_RETENTION_PERIOD , queueConfig.ErrorQueueRetentionPeriod.TotalSeconds.ToString(CultureInfo.InvariantCulture)},
```

Comment:
These `ToString()`s could be fixed as per your concern by specifying a format string that requires no decimal places, such as `"F0"`, which appears to do rounding rather than truncation.

Using [this](https://www.microsoft.com/net/learn/dotnet/in-browser-tutorial/1) for a quick REPL, the following output comes from this code:

```csharp
using System;

namespace Formatting
{
    internal static class Program
    {
        internal static void Main()
        {
            var duration = TimeSpan.FromSeconds(1234.56);
            Console.WriteLine(duration.TotalSeconds.ToString(System.Globalization.CultureInfo.InvariantCulture));
            Console.WriteLine(duration.TotalSeconds.ToString("N0", System.Globalization.CultureInfo.InvariantCulture));
            Console.WriteLine(duration.TotalSeconds.ToString("F0", System.Globalization.CultureInfo.InvariantCulture));
        }
    }
}
```

```sh
1234.56
1,235
1235
```